Sources/Sentry/include/SentryTracer.h

```diff
@@ -103,6 +103,11 @@ static NSTimeInterval const SentryTracerDefaultTimeout = 3.0;
  */
 @property (nullable, nonatomic, weak) id<SentryTracerDelegate> delegate;
 
+/**
+ * Indicates whether the event created with this tracer needs to include debug images info.
+ */
+@property (nonatomic) BOOL requiresDebugMeta;
```

Comment:
`m`: I think we can remove this property. Why don't we check if data contains `@"call_stack"` and then add the debug meta. I'm not a huge fan of setting boolean flags for statuses that can already be checked depending on some existing information.

Reply:
What if the user created a call_stack data themself?